src/helpers/ens.ts

```diff
@@ -4,7 +4,7 @@ import { resolveContent } from '@snapshot-labs/snapshot.js/src/utils/contentHash
 import gateways from '@snapshot-labs/snapshot.js/src/gateways.json';
 import utils from '@snapshot-labs/snapshot.js/src/utils';
 
-const gateway = process.env.VUE_APP_IPFS_GATEWAY || gateways[0];
+const gateway = import.meta.env.VITE_APP_IPFS_GATEWAY || gateways[0];
```

Comment:
Does this need the `VITE_APP` or can't we just do `IPFS_GATEWAY` now that we are using `import.meta`

Reply:
Must be VITE_APP AFAIK

Reply:
Actually not, i've just removed APP prefix